examples/producer-consumer/producer.pony

```diff
@@ -0,0 +1,23 @@
+use "collections"
+
+actor Producer
+	var _quantity_to_produce: U32
+	let _buffer: Buffer
+	var _num: U32 = 0
+	let _env: Env
+	
+	new create(quantity_to_produce: U32, buffer: Buffer, env: Env) =>
```

Comment:
rather than passing in env, you can pass in just the StdStream that represents 'env.out' as it is all you are using.